### PR TITLE
Use generate informers for cluster resource quota

### PR DESCRIPTION
**Pull request by smarterclayton**
Builds on the other changes to remove handwritten informers, only the 
last commit is new.

[test] @deads2k

### CODE REVIEW

File: pkg/cmd/server/origin/master.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func (c *MasterConfig) newOpenshiftAPIConfig(kubeAPIServerConfig apiserver.Confi
 		KubeletClientConfig:                c.KubeletClientConfig,
 		KubeInternalInformers:              c.Informers.InternalKubernetesInformers(),
 		AuthorizationInformers:             c.AuthorizationInformers,
-		DeprecatedInformers:                c.Informers,
+		QuotaInformers:                     c.QuotaInformers,
```
Comment (deads2k) [inline pkg/cmd/server/origin/master.go]: makes me happy.

File: pkg/controller/shared/shared_informer.go [hunk 2]
```diff
@@ -97,18 +90,6 @@ func (f *sharedInformerFactory) Start(stopCh <-chan struct{}) {
 	}
 }
 
-func (f *sharedInformerFactory) StartCore(stopCh <-chan struct{}) {
```
Comment (deads2k) [inline pkg/controller/shared/shared_informer.go]: yay!

**Comment by deads2k** [comment]
lgtm [merge]

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
Unable to tell what flaked

**Comment by smarterclayton** [comment]
```
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.TestCancelAndReadd (from github.com_openshift_origin_vendor_k8s.io_kubernetes_pkg_controller_node)

Failing for the past 1 build (Since Failed#2217 )
Took 0 ms.
add description
Stacktrace

=== RUN   TestCancelAndReadd
panic: sync: negative WaitGroup counter

goroutine 402 [running]:
panic(0x23db480, 0xc4204636e0)
	/usr/local/go/src/runtime/panic.go:500 +0x1ae
sync.(*WaitGroup).Add(0xc4203e28b0, 0xffffffffffffffff)
	/usr/local/go/src/sync/waitgroup.go:75 +0x27c
sync.(*WaitGroup).Done(0xc4203e28b0)
	/usr/local/go/src/sync/waitgroup.go:100 +0x42
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.TestCancelAndReadd.func1(0xc4204f6c60, 0xc400000001, 0xc400000020)
	/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node/timed_workers_test.go:118 +0x65
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.(*TimedWorkerQueue).getWrappedWorkerFunc.func1(0xc4204f6c60, 0x0, 0x0)
	github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node/_test/_obj_test/timed_workers.go:90 +0xb6
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.CreateWorker.func1()
```

**Comment by smarterclayton** [comment]
[severity:bug]

**Comment by smarterclayton** [comment]
[test]

**Pull request merged**